eng/Versions.props

```diff
@@ -180,9 +180,9 @@
     <BenchmarkDotNetPackageVersion>0.10.13</BenchmarkDotNetPackageVersion>
     <CastleCorePackageVersion>4.2.1</CastleCorePackageVersion>
     <FSharpCorePackageVersion>4.2.1</FSharpCorePackageVersion>
-    <GoogleProtobufPackageVersion>3.7.0</GoogleProtobufPackageVersion>
-    <GrpcAspNetCoreServerPackageVersion>0.1.21-dev201905010701</GrpcAspNetCoreServerPackageVersion>
-    <GrpcToolsPackageVersion>1.20.0-pre3</GrpcToolsPackageVersion>
+    <GoogleProtobufPackageVersion>3.8.0</GoogleProtobufPackageVersion>
+    <GrpcAspNetCoreServerPackageVersion>0.1.21-dev201906020701</GrpcAspNetCoreServerPackageVersion>
```

Comment:
Note that this package version is temporarily the latest nightly build, and is hosted on gRPC's own package feed.

We will get a version published to NuGet very soon and update this PR.